xml/System.Threading/Timer.xml

```diff
@@ -716,8 +716,8 @@ Sub TimerCallback(state As Object)
         <Parameter Name="period" Type="System.TimeSpan" />
       </Parameters>
       <Docs>
-        <param name="dueTime">A <see cref="T:System.TimeSpan" /> representing the amount of time to delay before invoking the callback method specified when the <see cref="T:System.Threading.Timer" /> was constructed. Specify negative one (-1) milliseconds to prevent the timer from restarting. Specify zero (0) to restart the timer immediately.</param>
-        <param name="period">The time interval between invocations of the callback method specified when the <see cref="T:System.Threading.Timer" /> was constructed. Specify negative one (-1) milliseconds to disable periodic signaling.</param>
+        <param name="dueTime">A <see cref="T:System.TimeSpan" /> representing the amount of time to delay before invoking the callback method specified when the <see cref="T:System.Threading.Timer" /> was constructed. Specify <see cref="F:System.Threading.Timeout.InfiniteTimeSpan" /> milliseconds to prevent the timer from restarting. Specify <see cref="F:System.TimeSpan.Zero" /> to restart the timer immediately.</param>
```

Comment:
```suggestion
        <param name="dueTime">A <see cref="T:System.TimeSpan" /> representing the amount of time to delay before invoking the callback method specified when the <see cref="T:System.Threading.Timer" /> was constructed. Specify <see cref="F:System.Threading.Timeout.InfiniteTimeSpan" /> to prevent the timer from restarting. Specify <see cref="F:System.TimeSpan.Zero" /> to restart the timer immediately.</param>
```

Reply:
done. Sorry